x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/index/mapper/GeoShapeWithDocValuesFieldTypeTests.java

```diff
@@ -161,38 +215,12 @@ public void testFetchSourceValueDateLine() throws IOException {
             false,
             geoFormatterFactory
         ).build(MapperBuilderContext.ROOT).fieldType();
-        // Test a polygon crossing the dateline
-        Object sourceValue = "POLYGON((170 -10, -170 -10, -170 10, 170 10, 170 -10))";
-        String polygonDateLine = "MULTIPOLYGON (((180.0 -10.0, 180.0 10.0, 170.0 10.0, 170.0 -10.0, 180.0 -10.0)),"
-            + "((-180.0 10.0, -180.0 -10.0, -170.0 -10.0, -170.0 10.0, -180.0 10.0)))";
-        Map<String, Object> jsonPolygonDateLine = Map.of(
-            "type",
-            "MultiPolygon",
-            "coordinates",
-            List.of(
-                List.of(
-                    List.of(List.of(180.0, -10.0), List.of(180.0, 10.0), List.of(170.0, 10.0), List.of(170.0, -10.0), List.of(180.0, -10.0))
-                ),
-                List.of(
-                    List.of(
-                        List.of(-180.0, 10.0),
-                        List.of(-180.0, -10.0),
-                        List.of(-170.0, -10.0),
-                        List.of(-170.0, 10.0),
-                        List.of(-180.0, 10.0)
-                    )
-                )
-            )
-        );
-
-        assertEquals(List.of(jsonPolygonDateLine), fetchSourceValue(mapper, sourceValue, null));
-        assertEquals(List.of(polygonDateLine), fetchSourceValue(mapper, sourceValue, "wkt"));
 
-        String mvtPolygonDateLine = "MULTIPOLYGON (((180.0 -10.0, 180.0 10.0, 170.0 10.0, 170.0 -10.0, 180.0 -10.0)),"
-            + "((-180.0 10.0, -180.0 -10.0, -170.0 -10.0, -170.0 10.0, -180.0 10.0)))";
+        assertEquals(List.of(jsonValue), fetchSourceValue(mapper, sourceValue, null));
+        assertEquals(List.of(wktValue), fetchSourceValue(mapper, sourceValue, "wkt"));
 
-        List<?> mvtExpected = fetchSourceValue(mapper, mvtPolygonDateLine, "mvt(0/0/0@256)");
-        List<?> mvt = fetchSourceValue(mapper, sourceValue, "mvt(0/0/0@256)");
+        List<?> mvtExpected = fetchSourceValue(mapper, mvtEquivalentAsWKT, "mvt(0/0/0@4096)");
+        List<?> mvt = fetchSourceValue(mapper, sourceValue, "mvt(0/0/0@4096)");
```

Comment:
The tests still pass with the original `@256`, what was the reason for changing to `@4096`? I presume it was just to match the default value?

Reply:
that's weird, I changed it so the extent is random.